master/buildbot/test/fake/docker.py

```diff
@@ -48,9 +48,11 @@ def stop(self, id):
     def wait(self, id):
         return 0
 
-    def build(self, fileobj, tag):
+    def build(self, fileobj, pull, tag):
         if fileobj.read() == b'BUG':
             pass
+        elif pull != bool(pull):
```

Comment:
not sure what this is about

Reply:
I think it was complaining that `pull` is unused or some other complaint. This seemed like the cleanest fix, if the value is valid, it will pretend an image is created, otherwise it will pretend it failed to create an image. At least that's how I understand it.